doc/plugins.md

```diff
@@ -64,6 +64,9 @@ The list of plugins:
   (👉 **note**: alternative to [`remark-capitalize`](https://github.com/zeit/remark-capitalize))
 * 🟢 [`remark-cite`](https://github.com/benrbray/remark-cite)
   – new syntax for Pandoc-style citations
+* 🟢 [`remark-cjk-friendly` / `remark-cjk-friendly-gfm-strikethrough`](https://github.com/tats-u/markdown-cjk-friendly)
+  – mitigates the problem in CommonMark/GFM that emphasis or
+  strikethrough are sometimes not recognized as intended in CJK languages
```

Comment:
Please use separate entries: you are adding 2 plugins, so please add 2 bullets too.
You can use *deep* links into your monorepo so that each link points to the exact docs.

Also, can you use shorter descriptions? Try to explain to the reader why they should (not) use your plugin, as quickly as possible.

Thank you!

Reply:
I split the entry and made the description so short that `node --run format` complains nothing.